pyproject.toml

```diff
@@ -81,3 +81,8 @@ exclude_lines = [
 ignore = [
     "E501",  # Ignore line length errors.
 ]
+
+[tool.pytest.ini_options]
+pythonpath = [
```

Comment:
Added this so you can run `pytest` instead of `python -m pytest`. Also makes things like VScode's pytest test discovery work properly.

Reply:
Interesting, I always run `pytest` directly from the repository root and have never had issues. Is this behavior different from what you see? Or are we running things differently?

(I also don't use VS Code, but making the project settings compatible with it seems reasonable)

Reply:
I would just get `ImprotError` for not being able to import `tyro`. Perhaps you installed tyro in editable mode, i.e., `pip install -e .`?

Reply:
Oh yeah, I have a habit of keeping the package installed when developing. Error makes sense if you don't.